core/commands/object.go

```diff
@@ -344,39 +344,21 @@ Data should be in the format specified by the --inputenc flag.
 }
 
 // objectData takes a key string and writes out the raw bytes of that node (if there is one)
-func objectData(n *core.IpfsNode, fpath path.Path) (io.Reader, error) {
-	dagnode, err := core.Resolve(n, fpath)
+func objectData(n *dag.Node, err error) (io.Reader, error) {
 	if err != nil {
 		return nil, err
 	}
 
-	log.Debugf("objectData: found dagnode %s (# of bytes: %d - # links: %d)", fpath, len(dagnode.Data), len(dagnode.Links))
-
-	return bytes.NewReader(dagnode.Data), nil
+	return bytes.NewReader(n.Data), nil
 }
 
 // objectLinks takes a key string and lists the links it points to
-func objectLinks(n *core.IpfsNode, fpath path.Path) (*Object, error) {
-	dagnode, err := core.Resolve(n, fpath)
+func objectLinks(n *dag.Node, err error) (*Object, error) {
 	if err != nil {
 		return nil, err
 	}
 
-	log.Debugf("objectLinks: found dagnode %s (# of bytes: %d - # links: %d)", fpath, len(dagnode.Data), len(dagnode.Links))
-
-	return getOutput(dagnode)
-}
-
-// objectGet takes a key string from args and a format option and serializes the dagnode to that format
-func objectGet(n *core.IpfsNode, fpath path.Path) (*dag.Node, error) {
-	dagnode, err := core.Resolve(n, fpath)
-	if err != nil {
-		return nil, err
-	}
-
-	log.Debugf("objectGet: found dagnode %s (# of bytes: %d - # links: %d)", fpath, len(dagnode.Data), len(dagnode.Links))
-
-	return dagnode, nil
+	return getOutput(n)
 }
 
```

Comment:
I would had to add `ctx`to each of the above transform functions and then pass it on to `core.Resolve()` so I rewrote these to just take the return value of it.

`objectGet` did nothing but debug print. If these prints are wanted/needed, I can write it the old way.